### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JaimeRZP**

### CODE REVIEW

**Comment by JaimeRZP** [comment]
@carlosggarcia  @damonge Could you have a look and merge if you are happy with the new readme?

File: README.md [hunk 1]
```diff
@@ -1,14 +1,103 @@
-# xCell
+![](https://raw.githubusercontent.com/JaimeRZP/Cosmoteka_tutorials/master/docs/src/assets/cosmoteka_logo.png)
 [![Coverage Status](https://coveralls.io/repos/github/xC-ell/xCell/badge.svg?branch=master)](https://coveralls.io/github/xC-ell/xCell?branch=master)
```
Comment (carlosggarcia) [inline README.md:2]: I'd put the badges at the top, but not strong feelings.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,103 @@
...
 
-Pipeline to compute Nx2pt angular power spectra and their covariances.
+Welcome to the Cosmoteka, the largest repository of consistently combined angular power spectra in Cosmology! 
 
-# Usage
-In order to run the code use `python3 run_cls.py input/kv450_1024.yml cls`.
```
Comment (carlosggarcia) [inline README.md:7]: The link to the tutorials is good, but I'd leave a example of how to run this in the terminal. In the end, the idea is to run this stuff in parallel in a cluster, so better leave a comment on this.

File: README.md [hunk 3]
```diff
@@ -1,14 +1,103 @@
...
+| ```KiDS1000```    | Weak lensing                      | [Heymans et al, 2020](https://arxiv.org/abs/2007.15632)               | [catalogue](https://kids.strw.leidenuniv.nl/DR4/data_files/)                                      | D. Alonso              |
+| ```NVSS```        | Galaxy clustering                 | [Condon et al, 1998](https://iopscience.iop.org/article/10.1086/300337/pdf) | [catalogue](https://www.cv.nrao.edu/nvss/)                                                  | F. Oliveira-Franco     |
+| ```P15tSZ```      | Compton-y map                     | [Planck Collaboration, 2015](https://arxiv.org/abs/1502.05956)        | [catalogue](https://irsa.ipac.caltech.edu/data/Planck/release_2/all-sky-maps/ysz_index.html)      | J. Ruiz-Zapatero       |
+| ```P18CMBk```     | CMB convergence                   | [Planck collaboration, 2018](https://arxiv.org/abs/1807.06210)        |                                                                                                   | J. Ruiz-Zapatero       |
```
Comment (carlosggarcia) [inline README.md:31]: Planck Lensing got from, I think: http://pla.esac.esa.int/pla/aio/product-action?COSMOLOGY.FILE_ID=COM_Lensing_4096_R3.00.tgz (@damonge, check)

File: README.md [hunk 4]
```diff
@@ -1,14 +1,103 @@
...
+| ```SPT```         | Compton-y map                     | [Bleem et al, 2021](https://arxiv.org/abs/2102.05033)                 | [catalogue](https://lambda.gsfc.nasa.gov/product/spt/spt_prod_table.html)                         | J. Ruiz-Zapatero       |
+| ```WIxSC```       | Galaxy clustering                 | [Bilicki et al, 2016](https://arxiv.org/abs/1607.01182)               | [catalogue](http://ssa.roe.ac.uk/WISExSCOS.html)                                                  | D. Alonso              |
 
---- 
-More info about the sacc files in https://github.com/LSSTDESC/sacc
+# Design Philosophy
+![](https://raw.githubusercontent.com/JaimeRZP/Cosmoteka_tutorials/master/docs/src/assets/Cosmoteka_schematic_v2.png)
+
+Cosmoteka is a large collection of pipelines to process map-level data from a wide variaty of surveys into angular power spectra combined in a statistically consistent manner. Cosmoteka accomplishes this by computing all the cross- and auto-covariances of the different maps.
```
Comment (carlosggarcia) [inline README.md:40]:
```
Cosmoteka is a single pipeline that processes catalog level data from a wide variety of surveys and measures their angular power spectra and covariances in a consistent manner. Cosmoteka heavily relies on `NaMaster`.
```

File: README.md [hunk 5]
```diff
@@ -1,14 +1,103 @@
...
+
+Cosmoteka is designed to allow for the largest amount of modularity possible to encourage open-source community development. Inside of each module, Cosmoteka follows an object-oriented approach. Thus, given some configureation file, each module defines a series of objects which host the methods used to process the data at each step of the pipeline.
```
Comment (carlosggarcia) [inline README.md:42]:
```
Thus, given a configuration file or dictionary, the user can instantiate a class that constains all the methods needed to create a sky map (mappers), to estimate the angular power spectrum of two fields (cl) or the covariance (cov).
```

File: README.md [hunk 6]
```diff
@@ -1,14 +1,103 @@
...
...the `mapper` modules compute the `NaMaster` fields we can look the `mapper_base.py`, the mapper module which defines the parent class for all other mappers inside Cosmoteka. As the parent class, `mapper_base` defines a series of default methods which are then overwritten by the child mappers to perform the data processing specific to each data set. Moreover, it also defines common methods to each mapper. The most important of these methods is `get_nmt_field` which returns the `NaMaster` field from each mapper. Inside `get_nmt_field`, the signal map and mask of each mapper are computed according to the specifc methods of the children mappers. Then, the signal map and mask as well as other mapper specific quantities (data beam, contaminants,... etc) are passed on to `NaMaster` to compute the final field. Once the `NaMaster` fields are computed, `cls` orchestrates the computation of the angular power spectra and their covariance requested by the user. A brief description of the role of each module inside `cls` can be found in the table below. For a more detailed description of the role of the  `cls` modules as well as the individual mappers please visit the documentation of Cosmoteka.
```
Comment (carlosggarcia) [inline README.md:44]: I find this a bit confusing and too long. I would just let the table below talk by itself. Add  `mappers` there and explain what you say here: they project the catalogs into sky maps and provide `NmtField`s.

File: README.md [hunk 7]
```diff
@@ -1,14 +1,103 @@
...
+
+| Module            | function                                                                                                               |
+| -----------       | :-----------                                                                                                           |
```
Comment (carlosggarcia) [inline README.md:47]: What I said before, add here the mappers explanation

File: README.md [hunk 8]
```diff
@@ -1,14 +1,103 @@
...
+| ```cl.py```       | Computes the Cl's requested by the user in the configuration file from the `NaMaster` fields provided by the mappers.  |
+| ```cov.py```      | Uses the theoretical predictions of ```theory.py``` to compute the covariance of the angular power spectra.            |
```
Comment (carlosggarcia) [inline README.md:49]: We can also use data directly to estimate the covariance

File: README.md [hunk 9]
```diff
@@ -1,14 +1,103 @@
...
+| ```data.py```     | Reads the user configuration file and reaches out to the relevant mappers.                                             |
```
Comment (carlosggarcia) [inline README.md:50]: `reaches out to the relevant mappers` -> `can return an instance of the relevant mappers`

File: README.md [hunk 10]
```diff
@@ -1,14 +1,103 @@
...
+| ```theory.py```   | Computes the a theory prediction for the Cl's computed in  `cl.py` using  `pyccl`.                                     |
```
Comment (carlosggarcia) [inline README.md:51]: I would add `(only available for some observables)`

**Review by carlosggarcia — CHANGES_REQUESTED** [review]
Great work! Some extra comments:
 - We can compute the covariance directly from data, without using a fiducial model, so I would split that step into two options:
  -- cl.py ------------------> cov.py
  -- cl.py -> theory.py -> cov.py
 - I think the references given are mixing the data release papers and the cosmology papers. I would cite the later. E.g. HSC (but I see that it's cited inside the mapper, so @damonge?)
 - KV450 and SMICA mappers are missing.
 - Arxiv -> arXiv

File: README.md (hunk 7 above)
Comment (carlosggarcia) [reply README.md:47]: Same as before, only about the table.

File: README.md [hunk 11]
```diff
@@ -1,14 +1,103 @@
...
+| ```DESY3wl```     | Weak lensing                      | [Abbot et al, 2021](https://arxiv.org/abs/2105.13549)                 | [catalogue](https://desdr-server.ncsa.illinois.edu/despublic/y3a2_files/)                         | C. Garcia-Garcia       |
+| ```eBOSS```       | Quasar clustering                 | [Gil-Marin et al, 2018](https://arxiv.org/abs/1801.02689)             | [cataloge](https://data.sdss.org/sas/dr14/eboss/)                                                 | J. Ruiz-Zapatero       |
+| ```HSC_DR1wl```   | Weak lensing                      | [Aihara et al, 2017](https://arxiv.org/abs/1702.08449)                | [catalogue](https://hsc-release.mtk.nao.ac.jp/doc/index.php/tools/)                               | D. Alonso              |
+| ```KiDS1000```    | Weak lensing                      | [Heymans et al, 2020](https://arxiv.org/abs/2007.15632)               | [catalogue](https://kids.strw.leidenuniv.nl/DR4/data_files/)                                      | D. Alonso              |
```
Comment (carlosggarcia) [inline README.md:28]: Maybe we should cite this one: https://arxiv.org/pdf/2007.15633.pdf?

Comment (carlosggarcia) [reply README.md:28]: or this one? https://arxiv.org/pdf/2007.01845.pdf

File: README.md (hunk 3 above)
Comment (JaimeRZP) [reply README.md:31]: Added!

File: README.md (hunk 1 above)
Comment (JaimeRZP) [reply README.md:1]: Done!

File: README.md (hunk 2 above)
Comment (JaimeRZP) [reply README.md:7]: Brought it back!

File: README.md (hunk 4 above)
Comment (JaimeRZP) [reply README.md:40]: Done!

File: README.md (hunk 8 above)
Comment (JaimeRZP) [reply README.md]: Added!

File: README.md (hunk 10 above)
Comment (JaimeRZP) [reply README.md]: Done!

File: README.md (hunk 5 above)
Comment (JaimeRZP) [reply README.md:42]: Done!

File: README.md (hunk 9 above)
Comment (JaimeRZP) [reply README.md]: Done!

File: README.md (hunk 11 above)
Comment (JaimeRZP) [reply README.md]: Cited the catalog paper instead

File: README.md (hunk 9 above)
Comment (carlosggarcia) [reply README.md]: I see it unchanged xD

Comment (JaimeRZP) [reply README.md]: Soz should be changed now!

**Review by carlosggarcia — APPROVED** [review]
LGTM!

**Pull request merged**